Jenkinsfile

```diff
@@ -90,25 +96,25 @@ def buildGrakn() {
 
 //Only run validation master/stable
 if (env.BRANCH_NAME in ['master', 'stable']) {
-    properties([
-	buildDiscarder(logRotator(numToKeepStr: '30', artifactNumToKeepStr: '7'))
-    ])
+    properties([buildDiscarder(logRotator(numToKeepStr: '30', artifactNumToKeepStr: '7'))])
     node {
-	String workspace = pwd()
-	checkout scm
+        String workspace = pwd()
```

Comment:
spaces and tabs in the same file 😐

Reply:
...this is me removing the tabs 👍 
now it's all spaces.

Reply:
_stares suspiciously at you_ 😐